common/lib/xmodule/xmodule/tests/test_import.py

```diff
@@ -570,7 +570,7 @@ def test_unicode(self):
         ]
 
         self.assertTrue(any(
-            expect in msg or expect in err
+            expect in msg.decode('utf-8') or expect in err.decode('utf-8')
```

Comment:
`msg` and `err` were encoded to UTF-8 in the code immediately before this, and aren't used again afterwards; can we avoid this encode/decode round trip?